frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -233,7 +233,7 @@ export function Experiments(): JSX.Element {
                                             }
                                         }}
                                         options={[
-                                            { label: 'All', value: ProgressStatus.All },
+                                            { label: 'All', value: 'all' },
```

Comment:
Where/why did this go? 🤔

Reply:
"All" is not not an actual status, it's only a filtering option, which was making handling of actual experiment items weird, because TypeScript said "you need to handle the All status too" – despite it not really existing. It should just be a string here.